### DISCUSSION TITLE
[milvus v0.11.0] SQLite does not have Tables or TableFiles tables to get information about the index

### DISCUSSION THREAD

**Discussion by deil87** [milvus-io/milvus#4467] [Q&A and General discussion] [answered]
Original intent was to get index's creation progress/status. According to the following answer https://github.com/milvus-io/milvus/discussions/3963 this information could be found in a metadata table(s) (sqlite in my case).

Here is a documentation page that shows some examples of how to retrieve `engine_type` field along with the other index related fields https://www.milvus.io/cn/blogs/2019-12-24-view-metadata.md

Unfortunately I was not able to find those tables. I've checked all available tables and it seems they don't contain information about index as well.

Here is the output of the `sqlite > .tables` command:

![image](https://user-images.githubusercontent.com/1162131/102333265-78873e80-3f8d-11eb-91e0-6e363b80825b.png)

According to `.getCollectionInfo()` method index does exist:

![image](https://user-images.githubusercontent.com/1162131/102333520-d61b8b00-3f8d-11eb-9a39-f796b65ae53a.png)

Any ideas where I can find index related information?

**Comment by shiyu22** [comment, marked as answer]
Hi, actually before Milvus 0.11.0 you can use SQLite to get Tables and TableFiles, with 0.11 you can use [get_collection_info](https://pymilvus.readthedocs.io/en/0.3.0/api.html?highlight=index#milvus.Milvus.get_collection_info) to view index information

**Comment by yhmo** [comment]
In 0.10.x and old version, the meta schema has two tables: "Tables" and "TableFiles"
In 0.11.x, the meta schema is changed, become 11 tables as you see.
